pyscript.core/src/stdlib/pyscript.py

```diff
@@ -9,17 +9,19 @@
     from _pyscript.display import display
     from _pyscript import sync
 else:
-    # in MAIN both PyWorker and a runtime currentScript.id exist
-    # so these are both exposed and the display, if imported,
-    # will point at the right script as default target
-    PyWorker = window._pyscript.PyWorker
+    # in MAIN both PyWorker and current element target exist
+    # so these are both exposed and the display will use,
+    # if not specified otherwise, such current element target
+    import _pyscript_js
+    from _pyscript.display import display as py_display
```

Comment:
Maybe keep this private:
```suggestion
    from _pyscript.display import display as _py_display
```

Reply:
it's `_display` now, but I've changed that before reading this, I hope that's fine.